AUTHORS.txt

```diff
@@ -12,11 +12,17 @@
 # Please keep the list sorted.
 
 Amazon
+Apple
 Aurelius
+DataDog
```

Comment:
👋 Hey Boxuan. Adding DataDog to that list would require proper discussion and validation with Datadog representatives. Would you mind removing it from the list please?

Reply:
I removed `DataDog` from the list and force-pushed to Boxuan's branch

Reply:
Thank!

Reply:
Good point, maybe I should remove all companies’ names here.

Reply:
`Mapped` is fine to leave in the list. I assume as @farodin91 is approved then `G Data` is also fine.

Reply:
I am assuming @ntisseyre might say no to an addition here.

Meanwhile, I am seeking Microsoft’s approval.

Reply:
I didn’t get a concrete answer from my current employee so I’ll just remove it.

I’ll revert changes to this file and only leave addition of Mapped and G Data.